paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -65,6 +65,9 @@ abstract class PrepareResourcesTask : DefaultTask() {
 
   private val projectDirectory = project.layout.projectDirectory
 
+  private val gradleUserHomeDirectory =
+    project.layout.dir(project.provider { project.gradle.gradleUserHomeDir }).get()
```

Comment:
Does going through the provider buy you anything? There's an overload which takes a path string directly.
```suggestion
    project.layout.dir(project.gradle.gradleUserHomeDir.path)
```

Reply:
Oh nice, when reading the docs, it seemed like dir(String) would resolve the path relative to the project root directory, whereas the Provider wouldn't.  But if this works, I like this much better!

Reply:
actually, I don't see an overload for `ProjectLayout.dir(String file)`; did you mean `Project.getProjectDirectory().dir(String path)`?